Controls/DataVisualization/DataVisualization.UWP/Bulletgraph/RadBulletGraph.cs

```diff
@@ -613,9 +613,9 @@ private void EnsureAllValuesInRange()
 
                     this.SetValuesInRange(FeaturedMeasureStartValueProperty, startValue, endValue);
                     this.SetValuesInRange(ComparativeMeasureProperty, startValue, endValue);
-                    this.SetValuesInRange(ProjectedMeasureProperty, this.FeaturedMeasureStartValue, endValue);
-                    this.SetValuesInRange(FeaturedMeasureProperty, this.FeaturedMeasureStartValue, endValue);
-                    this.SetValuesInRange(FeaturedMeasureStartValueProperty, startValue, this.FeaturedMeasure);
+                    this.SetValuesInRange(ProjectedMeasureProperty, startValue, endValue);
+                    this.SetValuesInRange(FeaturedMeasureProperty, startValue, endValue);
+                    this.SetValuesInRange(FeaturedMeasureStartValueProperty, startValue, endValue);
 
```

Comment:
Both the ProjectedMeasure and the FeaturedMeasure properties should have as a minimum value the set FeatureMeasureStartValue property. 

Consider the following scenario. If you have set the FeaturedMeasureStartValue to 40 ,FeaturedMeasure to 20 and StartValue to 0. The expected result would be the FeaturedMeasure property to become 40 after the EnsureAllValuesInRange is called. With your changes the minimum value for the FeaturedMeasure will be the StartValue and thus the FeaturedMeasure will become 0 after the EnsureAllValuesInRange is called- this is not the expected result. The same is also valid for the ProjectedMeasure property - it is expected to have as a minimum value the one set for the FeaturedMeasureStartValue.

The logic for the `this.SetValuesInRange(FeaturedMeasureStartValueProperty, startValue, this.FeaturedMeasure);` is similar. The FeaturedMeasureStartValue could not have value bigger than the one set for the FeaturedMeasure. With the proposed changes it could have as maximum value the one set for the EndValue.